### DISCUSSION TITLE
What is the base line?

### DISCUSSION THREAD

**Discussion by acarlo79** [jeroenterheerdt/HAsmartirrigation#432] [General]
Hi,

First of all, I love the idea and how v2 looks like.
Now my very basic question: while I understand how the bucket goes up and down based on rain/evaporation and other factors, what is still unknown to me is how much water the tool is considering to be good for the lawn.

Let me explain better:

Lawncare people suggest to water the lawn about 1" per week, do you use that as base line to then add or subtract watering based on weather condition?

Thx!

**Comment by jeroenterheerdt** [comment]
there is no baseline. It uses the FAO-56 Penman-Monteith method, which is the method recommended by the Food and Agriculture Organisation of the United Nations (FAO) for estimating (ETo) for a short grass crop using limited meteorological data (see Allen et al, 1998).

See: https://pyeto.readthedocs.io/en/latest/fao56_penman_monteith.html

**Reply by acarlo79** [reply]
Thanks!
Waht about the max bucket? any guidance on what shall be left to? it sounds like 50mm should be more than enough but I wanted check what would be the use case to increase or reduce it.

Thx!

**Reply by jeroenterheerdt** [reply]
Whatever you feel comfortable with. It doesn't matter that much, apart from this. Let's say you had a week of rain and your bucket grew to 50. Now, it has been dry and everyday 5 is taken out of the bucket. That means it will take 11 days of dry weather until the bucket is less than 0 and you'd see irrigation again. If that's a timeline you are comfortable with only you can answer. So it's hard to say, it depends on what you want and the type of weather patterns you have.

**Comment by acarlo79** [comment]
Perfect! thx for clarify.

At the moment I am using OWM and 1 day forecast for the calculation. I will check how it will behave with rainy days and sunny spells.
Overall, I am very impressed with how flexible can be to optimize the watering, great job!
